index.js

```diff
@@ -55,12 +55,14 @@ function Quickbooks(consumerKey, consumerSecret, token, tokenSecret, realmId, us
  * @param callback
  */
 Quickbooks.prototype.createToken = function(card, callback) {
-  module.request(this, 'post', {
-    url: '/payments/tokens',
-    headers: {
-      'Company-Id': this.realmId
-    }
-  }, card, callback)
+  request({
```

Comment:
Why do you use the `request` module directly here, rather than going through `module.request`?

Reply:
currently `/payments/tokens` endpoint results error if `Authentication` header is passed.  That's why I saw it's a better choice to make a separate call to endpoint.

Reply:
Can we keep using the `module.request` method and check the url there to conditionally exclude the `Authorization` header? It has logging and error handling. Also, currently this will post to the live Intuit endpoint even if the user has created the Quickbooks object with sandbox = true.

Reply:
We can do that but will it fail with OAuth 1.0 headers also like when OAuth 2.0 header is passed?